### ISSUE TITLE
Create TSL specification for playing a single note G and two notes G and E (alternating)

### ISSUE THREAD

**Issue by Rheakk** [Barnard-PL-Labs/tsltools#9]
Update: Danielle and I achieved two tasks this week, the first was to create a TSL specification for playing a single note G, and then creating a TSL specification for playing two notes (alternating between G and E). For the one note specification, we were able to code up looking at the examples in the specs folder (Heating.tsl, Alarm.tsl), the second we tried to “sculpt from marble” but taking the Heating.tsl file, and molding that into our two-note specification. We were able to compile a code executable, however, we are not sure if it is achieving what we need. We created a state diagram in order to visually represent what we think the state machine is. There seems to be one more state than necessary (state 0), I think the program should only have two states. But we were not sure how to achieve this.

**Comment by nupurd89** [comment]
Update: I created a lambda which uploads the index.js file and tsl2tlsf executable and turns tsl code to tlsf. The test succeeds but returns unknown with an error as it says the binary file can't run. I spent a while researching how to solve this problem and it seems as as though you have to compile the binaries in docker or in amazon EC2 before uploading the binary in order for the lambda to find them (example ways outlined [here](https://medium.com/@mayanknayyar95/aws-lambda-and-custom-binaries-ead12ca5c62e) and [here](https://www.petewilcock.com/using-poppler-pdftotext-and-other-custom-binaries-on-aws-lambda/)). I wanted to double check if this was the right way to go about it at the meeting before continuing!

**Comment by dc3347** [comment]
Update: Rhea and I first were able to ensure the TSL specification for 1 note (G) could generate the proper code using the tsltools. We then worked on creating a TSL specification for 2 notes (E + G) using Heating.tsl as an example. We used the tsltools to generate a code executable and created a state diagram to represent the state machine, however are a bit unsure about the 4th "if" statement in the generated code – in what case would that [block of code](https://docs.google.com/document/d/12w7WpemlLqTaZeaZJfTEBcqkwUHxeR20GXFhTIgbcKM/edit?usp=sharing) be reached?
